### DISCUSSION TITLE
Forge?

### DISCUSSION THREAD

**Discussion by LordKiriYT** [IrisShaders/Iris#565] [General]
Any plans on getting this to work with sodium reforged?

**Comment by BeansBeefBroccoli** [comment]
[I don't think so](https://github.com/IrisShaders/Iris#but-wheres-the-forge-version).

**Comment by Xfrtrex** [comment]
no, forge is slower than fabric

**Comment by IMS212** [comment]
No, there are no plans to do so due to both technical and community reasons.
